xml/art_virtualization-best-practices.xml

```diff
@@ -3241,6 +3241,162 @@ LC_ALL=C PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
       </para>
 <screen>&prompt.root;<command>echo 1 > /sys/devices/system/cpu/cpu<replaceable>X</replaceable>/online</command></screen>
     </sect2>
+
+    <sect2 xml:id="sec-vm-guest-scsi-pr">
+      <title>SCSI3 persistent reservation (S3-PR) on a multipathed device</title>
+      <para>
+        SCSI persistent reservations allow restricting access to block devices
+        to specific initiators in a shared storage setup.  When implementing
```

Comment:
This is quite difficult for me to understand, so I assume it might be hard for customers too.

What is an 'initiator'?

Reply:
I can't figure out a simple to understand AND rigorous term to replace the 'initiator'.
Do you think I should explain a very basic scsi term in a vt best practice article?

Reply:
I think we should phrase it from the user perspective, not describe the elements of the SCSI protocol.

What does it mean for the user?

What about (fix as needed):

SCSI persistent reservations allow restricting access to block devices in a shared storage setup, to avoid improper multiple parallel accesses to the same physical device from software components on local or remote hosts, which could lead to device damage, data corruption.